src/androidTest/java/com/couchbase/lite/ManagerTest.java

```diff
@@ -5,6 +5,8 @@
 import com.couchbase.lite.mockserver.MockDispatcher;
 import com.couchbase.lite.mockserver.MockHelper;
 import com.couchbase.lite.replicator.Replication;
+import com.couchbase.lite.store.ForestDBStore;
```

Comment:
Currently ForestDBStore is in couchbase-lite-java-forestdb package. I believe importing ForestDBStore might cause problem if couchbase-lite-java-forestdb is not submodule during compile time.
Instead of import it, can you dynamically load class of `com.couchbase.lite.store.ForestDBStore`?